ui2/src/pages/login/QuickSetup.tsx

```diff
@@ -49,9 +48,17 @@ import { CreateEvent } from '@daml/ledger';
 
 type Offer = CustodianOffer | DistributorOffer | SettlementOffer | ExchangeOffer | MatchingOffer;
 
+enum ServiceKind {
+  CUSTODY = 'Custody',
+  LISTING = 'Listing',
+  TRADING = 'Trading',
+  MATCHING = 'Matching',
+  SETTLEMENT = 'Settlement',
+}
```

Comment:
the other ServiceKind enum doesn't work here anymore unless I continue to filter out services that don't apply to this modal. Thought it would be best to just curate a list of services local to Quick Setup for now.